### PR TITLE
fix: remove sidebar mode for mweb

### PR DESCRIPTION
**Pull request by raviteja83**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx [hunk 1]
```diff
@@ -35,10 +35,19 @@ export const ScreenshareLayout = ({ peers, onPageChange, onPageSize, edgeToEdge
   }, [activeSharePeer, peers, isMobile, layoutMode]);
 
   useEffect(() => {
-    if (layoutMode !== LayoutMode.SIDEBAR) {
-      setLayoutMode(LayoutMode.SIDEBAR);
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx:39]: It worked last time with the hasSidebar prop being set to false. Do we still need that?

File: packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx [hunk 2]
```diff
@@ -35,10 +35,19 @@ export const ScreenshareLayout = ({ peers, onPageChange, onPageSize, edgeToEdge
...
+    if (isMobile) {
+      setLayoutMode(LayoutMode.GALLERY);
+      return;
     }
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+    if (layoutMode === LayoutMode.SIDEBAR) {
+      return;
+    }
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx:44]: Is this needed? It probably won't rerender if prevState === new value

Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx:44]: it is for the cleanup to not run

File: packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx (hunk 1 above)
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/VideoLayouts/ScreenshareLayout.tsx:39]: on resize also it has to be handled hence thjs

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**